workbench/electron-builder-config.js

```diff
@@ -36,8 +36,12 @@ const config = {
       to: 'invest',
     },
     {
-      from: '../dist/Miniforge3',
-      to: 'Miniforge3',
+      from: '../dist/micromamba', // mac
+      to: 'micromamba',
+    },
+    {
+      from: '../dist/micromamba.exe', // windows
+      to: 'micromamba',
```

Comment:
```suggestion
      to: 'micromamba.exe',
```

Reply:
Added this in another commit before I saw this